packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -48,6 +48,7 @@ const semver = require('semver')
 const slugify = require('slugify')
 const generatorPkg = require('../package.json')
 const Handlebars = require('handlebars')
+const validatePackageName = require('validate-npm-package-name')
```

Comment:
does this library actually make network requests and check availability of npm package names that are not taken?

Reply:
Seems like it doesn't https://github.com/npm/validate-npm-package-name/blob/main/lib/index.js